sys/vane/ames.hoon

```diff
@@ -523,8 +142,9 @@
           ::
               rov
             |-  ^-  (unit lane)
-            :: XX check will
-            ?:  ?|  !=(our (sein:title her))
+            ::  XX jael scry
+            ::
+            ?:  ?|  !=(our (^sein:title her))
```

Comment:
It can't be trivially converted, because there's no `+sley` function in scope. I deferred because it's not a trivial refactoring. But this code only exists to support the %rove/%tend routing subscriptions used by :dns, and that is all being removed.